ITEM.tsv

```diff
@@ -3129,13 +3129,13 @@ ITEM_20150317_003128	Torn Bat Wings
 ITEM_20150317_003129	Green Ointment
 ITEM_20150317_003130	A Vubbe ointment that is used to treat their injured. It's not possible to use it as is.
 ITEM_20150317_003131	Royal Tomb Guardian Power Source
-ITEM_20150317_003132	A power source used by the Guardian for movement.
+ITEM_20150317_003132	A power source used by the Guardian that allows it to move.
```

Comment:
Don't think the original translation was implying that the  "Guardian" power source was what allowed to move with it. Rather the power source could be used for movement. The original translation should be fine.